olp-cpp-sdk-core/src/client/OlpClient.cpp

```diff
@@ -286,5 +305,158 @@ CancellationToken OlpClient::CallApi(
       [cancel_context]() { cancel_context->CancelOperation(); });
 }
 
+std::tuple<http::NetworkResponse, std::stringstream> SendRequest(
```

Comment:
This is why I was in favour of adding the shared_ptr<std::stringstream> to the NetworkResponse.
Should we do it now? A response is a response, inkl. response body.

Reply:
Yes, we should do it in separate ticket.

Reply:
Sounds good.